README.md

```diff
@@ -173,6 +176,21 @@ type SwarmNode struct {
     Name    string
     Address Address
 }
+
+type ServerInfo struct {
+	Name          string
+	NumContainers int
+	NumImages     int
+	DockerInfo    DockerInfo
```

Comment:
If `ServerInfo` is renamed to `Docker`, field could renamed to just `Info`.  So the the template looks like `Docker.Info`.